libs/partners/mongodb/langchain_mongodb/chat_message_histories.py

```diff
@@ -46,18 +58,20 @@ def __init__(
 
         self.db = self.client[database_name]
         self.collection = self.db[collection_name]
-        self.collection.create_index("SessionId")
+
+        if create_index:
+            self.collection.create_index(self.session_id_key, **index_kwargs)
```

Comment:
since `index_kwargs: Optional[Dict]` this will need a NoneType check. 
```suggestion

        if create_index:
            index_kwargs = index_kwargs or {}
            self.collection.create_index(self.session_id_key, **index_kwargs)
```